manifests/s/Saturneric/GpgFrontend/2.0.8/Saturneric.GpgFrontend.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Saturneric.GpgFrontend
+PackageVersion: 2.0.8
+PackageLocale: en-US
+Publisher: Saturneric

```

```suggestion
Publisher: Codesdream
```
This must match Control Panel.

---

Code Dream, or Codesdream (https://git.codesdream.com/) is a non-profit owned by Saturneric. I haven't found any reference to them as a publisher per se, only as a host for the repo (https://www.gpgfrontend.pub/#/overview?id=source-code). But if they're listed as the publisher under control panel and they must match, then I suppose this should be ok.